README.md

```diff
@@ -17,3 +17,21 @@ $ mvn package
 ```
 
 There will be a `.xar` file in the `target/` sub-folder.
+
+
+## Development / Manual Testing
+
+Simply run: `mvn -Pdev docker:start` to start a Docker environment of eXist-db on port 9090
+with the semver.xq package already deployed.
+
+As the Docker environment binds the files from the host filesystem, changes to the source code
+are reflected immediately in th running eXist-db environment.
```

Comment:
`th` -> `the`

Reply:
@joewiz Ah ha. Thanks, I have fixed and rebased that now